codeviz.py

```diff
@@ -29,16 +29,16 @@ def __init__(self, filename):
     def get_includes(self, filename):
         includes_re = re.compile(r'\s*#\s*include\s+["<](?P<file>.+?)[">]')
 
-        with open(filename, 'rt') as f:
-            data = f.read()
-
-        # Remove all comments
-        data = re.sub(r'/\*.*?\*/', '', data, flags=re.DOTALL)
-        data = re.sub(r'//.*', '', data)
-
-        includes = includes_re.findall(data)
-
-        return includes
+        try:
+            with open(filename, 'rt') as f:
+                    data = f.read()
```

Comment:
Please fix indentation, here, and line 40.

Also, it appears that the only part that needs to wrapped in the try-catch is the file open operation. 

Wrap the try statement around the data reading, and `return []` at the exception with the message.